README.rst

```diff
@@ -1,4 +1,4 @@
-|gitter|_ |travis|_ |appveyor|_ |zenodo|_ |github|_ |codeclimate|_
+|gitter|_ |travis|_ |appveyor|_ |zenodo|_ |github|_
```

Comment:
It seems that the PR is based on an older version of `README.rst` since it is accidentally undoing a number of previous changes.

Since your change is relatively small and isolated: Perhaps you can copy the line you'd like to change, reset `README.rst` to the version from the current master branch, and just paste in your change again?

Reply:
OK. I will try that!